drink_machines.lua

```diff
@@ -397,6 +417,9 @@ minetest.register_node('drinks:liquid_barrel', {
       if outputstack == 'vessels:glass_bottle' then
          drinks.drinks_liquid_sub(4, 'jbo', 128, pos)
       end
+      if outputstack == 'vessels:steel_bottle' then
```

Comment:
wow, someone needs to really:

1) learn `elseif` and why it matters
2) migrate this to a lookup table!